security-command-center/snippets/src/test/java/vtwo/AssetSecurityMarksIT.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo;
+
+// import static org.junit.Assert.assertThat;
+import static com.google.common.truth.Truth.assertThat;
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+
+import com.google.cloud.testing.junit4.MultipleAttemptsRule;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import vtwo.assets.AddDeleteSecurityMarks;
+import vtwo.assets.AddSecurityMarksToAssets;
+import vtwo.assets.DeleteAssetsSecurityMarks;
+
+@RunWith(JUnit4.class)
+public class AssetSecurityMarksIT {
+
+  private static final String ORGANIZATION_ID = System.getenv("SCC_PROJECT_ORG_ID");
+  private static String assetId;
+  private static ByteArrayOutputStream stdOut;
+
+  @Rule
+  public final MultipleAttemptsRule multipleAttemptsRule =
+      new MultipleAttemptsRule(3, 120000); // 2 minutes
```

Comment:
2 minutes per test is too long. please consider to reduce the maximal time of execution.